### ISSUE TITLE
String not highlighted in class static props

### ISSUE THREAD

**Issue by dozoisch** [babel/babel-sublime#106]
I would've expected `"MainApplication"` to be yellow like normal strings? Or am I missing something.

This is using Babel Monokai.

![image](https://cloud.githubusercontent.com/assets/4422516/7335791/19be3094-eba5-11e4-8b5c-b9482990f440.png)

**Comment by zertosh** [comment]
Just test text for me...

```js
class Name extends React.Component {
  static displayName = 'Name'

  static PropTypes = {
    value: React.PropTypes.number
  }

  static fetchData = function fetchData(a, b, c) {

  }

  get getter() {}

  set setter() {}

  method() {}

  render() {
    return(
      <div>
        JSXText
      </div>
    )
  }
}
```

**Comment by dozoisch** [comment]
This is what I get 

![image](https://cloud.githubusercontent.com/assets/4422516/7341306/7e469164-ec6d-11e4-856d-22b686ed34dc.png)

**Comment by zertosh** [comment]
Oh yeah I know - sorry - it's a tester text for me. I put it here to keep track of it somewhere. The issue is actually kinda big. I need to refactor classes but I won't be able to get to it until next week.

**Comment by dozoisch** [comment]
Ohh my bad! Misunderstood!

Thanks a lot and good job!

**Comment by mathieumg** [comment]
@zertosh  Any news on this? Any way to help you?

**Comment by zertosh** [comment]
Yeah - I'm working on re-write using [Sublime Text 3's new `sublime-syntax`](https://www.sublimetext.com/docs/3/syntax.html), but I'm swamped right now :(

**Comment by mathieumg** [comment]
Thanks for the update, and I know that feeling all to well... :sob:

**Comment by zertosh** [comment]
I just published v7.1.0, it should be on Package Control soon. It fixes this and ton of other issues.

![es7-props-fixed](https://cloud.githubusercontent.com/assets/830952/8642314/223822ec-28f0-11e5-9a16-a8ccbd97b147.png)

**Comment by dozoisch** [comment]
@zertosh Awesome! Look forward to use it!
